.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - README.md
+    tags:
+      - "v*.*.*"  
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+  linux-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          submodules: recursive
+
+      - name: Prepare Plugin
+        run: |
+          cd Plugins/play-unreal-plugin && make 
+
+      - name: Package For Windows,Mac
+        run: |
+          FILENAME="CronosPlayDemo.zip"
+          cd ..
```

Comment:
will this packaging work with the plugin submodule? should it first init the submodule + run its makefile to download those binaries?

Reply:
yes, it's submodule initialized and run make in `Prepare Plugin` step

Reply:
where is `make` being executed in this flow?